tests/test_autopay.py

```diff
@@ -1,22 +1,30 @@
+import brownie
 import pytest
 from brownie import accounts
 from brownie import chain
 from brownie.network.account import Account
 from eth_abi import encode_single
+from multicall import multicall
+from multicall.constants import MULTICALL2_ADDRESSES
+from multicall.constants import MULTICALL_ADDRESSES
+from multicall.constants import Network
 from telliot_core.apps.core import TelliotCore
 from telliot_core.utils.response import ResponseStatus
 from telliot_core.utils.timestamp import TimeStamp
 from web3 import Web3
 
 from telliot_feed_examples.queries.query_catalog import query_catalog
-from telliot_feed_examples.reporters.reporter_autopay_utils import (
-    autopay_suggested_report,
-)
+from telliot_feed_examples.reporters.reporter_autopay_utils import autopay_suggested_report
+from telliot_feed_examples.reporters.reporter_autopay_utils import get_feed_tip
 
 
 @pytest.mark.asyncio
 async def test_main(mumbai_test_cfg, mock_flex_contract, mock_autopay_contract, mock_token_contract):
     async with TelliotCore(config=mumbai_test_cfg) as core:
+        addy = brownie.multicall.deploy({"from": accounts[0]})
+        Network.Brownie = 1337
```

Comment:
Add comment to inform future contributors what this number means & what it's fore